src/components/ComboboxMenu.tsx

```diff
@@ -39,8 +39,8 @@ export const ComboboxMenu = <
   onItemSelected,
 
   title,
-  inputPlaceholder = 'Search…',
-  slotEmpty = 'No items found.',
+  inputPlaceholder = 'Search…', // TODO: i18n
```

Comment:
oh yeah I don't think we can use `useStringGetter` in a default 🤔 eh the easiest way to get around this is to probably just use `inputPlaceholder ?? useStringGetter({key: STRING_KEYS.SEARCH})` in the code below (or if it's used in > 1 place, assign it somewhere at the top)